dbmigrate/src/main/java/se/fortnox/reactivewizard/dbmigrate/LiquibaseMigrate.java

```diff
@@ -37,11 +40,20 @@
  */
 public class LiquibaseMigrate {
 
+    private static final Logger LOG = LoggerFactory.getLogger(LiquibaseMigrate.class);
+
     private final List<Liquibase> liquibaseList;
 
     @Inject
     public LiquibaseMigrate(LiquibaseConfig liquibaseConfig) throws LiquibaseException, IOException {
         JdbcConnection conn = new JdbcConnection(getConnection(liquibaseConfig));
+        if (!Strings.isNullOrEmpty(liquibaseConfig.getTimeZone())) {
+            try {
+                conn.createStatement().execute("SET TIME ZONE '%s'".formatted(liquibaseConfig.getTimeZone()));
```

Comment:
Does this approach not require us to always need to keep in mind to set the time zone in every single place we create a new connection?:thinking: